### DISCUSSION TITLE
How to have Pyraf working on a MBP M2 with OS 13.4?

### DISCUSSION THREAD

**Discussion by trappist1i** [iraf-community/iraf#309] [Q&A]
I installed the Iraf version called iraf-main from https://github.com/orgs/iraf-community/discussions/297
on my new Mac Book Pro with a M2 chip and the OS Ventura 13.4 
I then installed Pyraf using the command pip3 install pyraf. 
The installation of Iraf went fine.
Now, when I launch Pyraf, I get the following message : "DEPRECATION WARNING: The system version of Tk is deprecated and may be removed in a future release. Please don't rely on it. Set TK_SILENCE_DEPRECATION=1 to suppress this warning.", and then nothing.

What could I do to solve the issue and to have pyraf working well?

**Comment by olebole** [comment]
When I had quick access to a recent Mac, I could reproduce this. However, as I don't own one, it is difficult for me to debug and solve. I need to wait for the chance to pick one for 1-2 days and I have spare time to do this.

Alternatively, you could go through the initialization of PyRAF and check why it fails.

**Reply by yuanqastro1996** [reply]
I launch Pyraf, Everything appears correct. But when I epar *, the parameter setting window cannot be displayed. As same as you, I'm using an M2 chip and the OS Ventura 13.4/13/6.


#####
type pyraf show:
(base) yuanq@YUANQDEMACBOOK-PRO ~ % pyraf
setting terminal type to 'xgterm' ...

  Community IRAF V2.17.1

  This product includes results achieved by the IRAF64 project in 2006-
  2009 directed by Chisato Yamauchi (C-SODA/ISAS/JAXA).

  Welcome to IRAF.  To list the available commands, type ? or ??.  To get
  detailed information about a command, type `help <command>'.  To run  a
  command  or  load  a  package,  type  its name.   Type  `bye' to exit a
  package, or `logout' to get out  of the CL.

  Visit http://github.com/iraf-community/iraf/issues to report problems.

  ***  Using global login file:  /Users/yuanq/.iraf/login.cl
  The following commands or packages are currently defined:

clpackage/:
 clpackage/     language/       obsolete/       softools/       utilities/
 dataio/        lists/          plot/           system/
 images/        noao/           proto/          user/
PyRAF 2.2.1
Python/CL command line wrapper
  .help describes executive commands
-->
